### ISSUE TITLE
Swift compile error

### ISSUE THREAD

**Issue by dragonfire1119** [intergalacticspacehighway/react-native-ios-live-text#1]
When this library is removed, everything compiles successfully.

```
CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'react-native-live-text-image-view' from project 'Pods')
	EmitSwiftModule normal x86_64 (in target 'react-native-live-text-image-view' from project 'Pods')
	CompileSwift normal x86_64 **/react-native-live-text-image-view/ios/LiveTextImageViewViewManager.swift (in target 'react-native-live-text-image-view' from project 'Pods')
	CompileSwiftSources normal x86_64 com.apple.xcode.tools.swift.compiler (in target 'react-native-file-access' from project 'Pods')
```

Versions:

React Native: 0.69.6

**Comment by intergalacticspacehighway** [comment]
Hi @dragonfire1119 do you think this could be similar https://github.com/facebook/react-native/issues/28777#issuecomment-645081516? Can you try some solutions suggested in that issue? 😅 

Also, do you have other iOS native modules in your app? Do they work fine?

**Comment by dragonfire1119** [comment]
@intergalacticspacehighway I found that thread before posting, and nothing has helped so far. Before adding this library, everything worked fine and compiled.

I also tried adding a bridge header and received the same error.

Ok got better error output on Xcode:

- node_modules/react-native-live-text-image-view/ios/LiveTextImageViewViewManager.swift:23:35: error: cannot find type 'ImageAnalysisInteraction' in scope
- node_modules/react-native-live-text-image-view/ios/LiveTextImageViewViewManager.swift:29:34: error: cannot find 'ImageAnalyzer' in scope
- node_modules/react-native-live-text-image-view/ios/LiveTextImageViewViewManager.swift:55:33: error: cannot find 'ImageAnalyzer' in scope
- node_modules/react-native-live-text-image-view/ios/LiveTextImageViewViewManager.swift:62:67: error: cannot infer contextual base in reference to member 'automatic'

Thinking I need to be compiling with Xcode 14 to have access to VisionKit?
